ui/tests/integration/components/status-report-meta-table-test.ts

```diff
@@ -0,0 +1,49 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { getUnixTime, subDays } from 'date-fns';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | meta-table', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
```

Comment:
I’d be tempted to split this test in two: one for the happy path and one for the empty state.